packages/edit-site/src/components/sidebar-dataviews/style.scss

```diff
@@ -27,3 +27,8 @@
 		color: $white;
 	}
 }
+
+.edit-site-sidebar-dataviews {
+	margin-left: -$grid-unit-20;
+	margin-right: -$grid-unit-20;
+}
```

Comment:
The element with this CSS class is a parent of the element with the `.edit-site-sidebar-dataviews-dataview-item` class, so let's move this CSS [here](https://github.com/WordPress/gutenberg/pull/67384/files#diff-78833cb86b24921f76235fb7e680f682c4516dc18560ad67ea135ccfea85e3e2R9).